cmd/utils/flags.go

```diff
@@ -1633,6 +1644,12 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		}
 		cfg.Genesis = core.DefaultGoerliGenesisBlock()
 		SetDNSDiscoveryDefaults(cfg, params.GoerliGenesisHash)
+	case ctx.GlobalBool(ScrollAlphaFlag.Name):
+		if !ctx.GlobalIsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 534353
+		}
+		cfg.Genesis = core.DefaultScrollAlphaGenesisBlock()
+		// SetDNSDiscoveryDefaults(cfg, params.ScrollAlphaGenesisHash)
```

Comment:
Why do we close `SetDNSDiscoveryDefaults`?

Reply:
This is for DNS-based discovery using hosts like `goerli.ethdisco.net`. We don't have this.

Reply:
Now I understand this, after reading EIP-1459 (Node Discovery via DNS).